MapleServer2/Constants/RecvOp.cs

```diff
@@ -1,152 +1,152 @@
-            RANK = 0X0059,
-            REQUEST_SKIP_TUTORIAL = 0X005A,
-            REQUEST_ITEM_BREAK = 0X005B,
-            RIDE_CONSUME_EP = 0X005C,
-            REQUEST_ADDITIONAL_EFFECT = 0X005D,
-            RECALL_USER = 0X005E,
-            REQUEST_ITEM_ENCHANT = 0X005F,
-            BLACK_MARKET = 0X0060,
-            GAMBLE = 0X0061,
-            PVP = 0X0062,
-            MAID = 0X0063,
-            NEWS_NOTIFICATION = 0X0064,
-            SMART_RECOMMEND_BILLING = 0X0065,
-            SYSTEM_SHOP = 0X0066,
-            ATTEND_GIFT = 0X0067,
-            PC_BANG_BONUS = 0X0068,
-            REQUEST_MANUFACTURER = 0X0069,
-            REQUEST_USER_ENV = 0X006A,
-            CASH = 0X006B,
-            NAMETAG_SYMBOL = 0X006C,
-            REQUEST_MOVE_FIELD = 0X006D,
-            WAITING_TICKET = 0X006E,
-            MATCH_PARTY = 0X006F,
-            RECALL_SCROLL = 0X0070,
-            POTENTIAL_ABILITY = 0X0071,
-            ENCHANT_SCROLL = 0X0072,
-            GLOBAL_PORTAL = 0X0073,
-            FISHING = 0X0074,
-            PLAY_INSTRUMENT = 0X0075,
-            CHANGE_ATTRIBUTES = 0X0076,
-            CHANGE_ATTRIBUTES_SCROLL = 0X0077,
-            ITEM_ENCHANT_TRANSFORM = 0X0078,
-            REQUEST_PET = 0X0079,
-            REQUEST_PET_INVENTORY = 0X007A,
-            NOTICE_DIALOG = 0X007B,
-            SKILL_COMPACT_CONTROL = 0X007C,
-            BANWORD = 0X007D,
-            CHECK_CHAR_NAME = 0X007E,
-            PLATFORM_PROTECT_PACKET = 0X007F,
-            PLATFORM_ACCOUNT_SAFE = 0X0080,
-            STATE = 0X0081,
-            MESO_MARKET = 0X0082,
-            GLOBAL_FACTOR = 0X0083,
-            SMART_PUSH = 0X0084,
-            PLAY_ARCADE = 0X0085,
-            ENTER_EVENTFIELD = 0X0086,
-            CARD_REVERSE_GAME = 0X0087,
-            REQUEST_ITEM_LOCK = 0X0088,
-            ITEM_SOCKET_SYSTEM = 0X0089,
-            CHARACTER_ABILITY = 0X008A,
-            TUTORIAL = 0X008B,
-            ITEM_SOCKET_SCROLL = 0X008C,
-            CONSTRUCT_SHOP = 0X0094,
-            STAT_POINT = 0X009A,
-            ITEM_REPEAT = 0X009B,
-            REQUEST_TUTORIAL_ITEM = 0X009D,
+            NULL = 0x0000,
+            RESPONSE_VERSION = 0x0001,
+            RESPONSE_LOGIN = 0x0003,
+            RESPONSE_KEY = 0x0004,
+            PONG = 0x0006,
+            CHARACTER_MANAGEMENT = 0x0009,
+            RESPONSE_HEARTBEAT = 0x000A,
+            REQUEST_TIME_SYNC = 0x000B,
+            RESPONSE_CLIENTTICK_SYNC = 0x000C,
+            REQUEST_QUIT = 0x000D,
+            REQUEST_LOAD_UGC_MAP = 0x000F,
+            RESPONSE_FIELD_ENTER = 0x0010,
+            USER_CHAT = 0x0011,
+            USER_SYNC = 0x0012,
+            EMOTION = 0x0013,
+            REQUEST_SERVER_LIST = 0x0014,
+            REQUEST_ITEM_CREATE = 0x0015,
+            ITEM_EQUIP = 0x0016,
+            REQUEST_ITEM_INVENTORY = 0x0017,
+            REQUEST_ITEM_STORAGE = 0x0018,
+            REQUEST_ITEM_USE = 0x0019,
+            ITEM_ExTRA_DATA = 0x001A,
```

Comment:
Uppercase x?

Reply:
Weird artifact from merging, I actually changed that to uppercase X but for some reason git decided that I did the opposite.